### PR TITLE
Increase pageSize for search result view

### PR DESCRIPTION
**Pull request by petergrau**
## Hey, I just made a Pull Request!

Increases the defaultPage size value from `5` to `20` for the search result page.

image

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Review by emmaindal — APPROVED** [review]
@petergrau Nicee, thank you 🎉

**Comment by emmaindal** [comment]
@backstage/maintainers need your approval because of changeset... 😸

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**
